ReportUtilityBills/tests/Sample Flat Rate Min Annual Charge.json

```diff
@@ -0,0 +1,649 @@
+{
+  "items": [
+    {
+      "name": "Sample Tiered Rate",
+      "sector": "Residential",
+      "description": "Sample file defined by BEopt.",
+      "fixedchargeunits": "$/month",
+      "fixedchargefirstmeter": 8,
+      "minchargeunits": "$/year",
+      "mincharge": 10000,
```

Comment:
I'm debating whether to make the min charge values (annual and monthly) more realistic. This is a note to self.